share/tools/web_config/sample_prompts/robbyrussell.fish

```diff
@@ -6,7 +6,12 @@ function fish_prompt
     if not set -q -g __fish_robbyrussell_functions_defined
         set -g __fish_robbyrussell_functions_defined
         function _git_branch_name
-            echo (git symbolic-ref HEAD ^/dev/null | sed -e 's|^refs/heads/||')
+            set -l branch (git symbolic-ref --quiet HEAD ^/dev/null)
+            if test $branch
```

Comment:
Please do not rely on `test some-arg` without a flag to specify the test to be performed. This is a POSIX bogosity that confuses people and results in bugs. Use either `test -n "$branch"` or, preferably, the fish idiom: `set -q branch[1]`.